app/src/main/java/com/gophillygo/app/activities/FilterableListActivity.java

```diff
@@ -52,16 +56,18 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // set up filter button
         filter = new Filter();
-        filterButton = findViewById(filterButtonId);
+        filterButton = findViewById(R.id.filter_bar_filter_button);
         filterButton.setOnClickListener(v -> {
-            FilterDialog filterDialog = FilterDialog.newInstance(filter);
+            FilterDialog filterDialog = FilterDialog.newInstance(new Filter(filter));
```

Comment:
This was done because the same filter object was being shared between both the `FilterDialog` and the activities, and updating the "Want To Go" or "Liked" buttons in the filter was toggling the toolbar buttons :fearful: 

I'll add a comment to that affect before merging.